make/docker-compose.clair.yml

```diff
@@ -17,7 +17,7 @@ services:
         aliases:
           - postgres
     container_name: clair-db
-    image: postgres:9.6.3
+    image: vmware/harbor-postgresql:9.6.3
```

Comment:
This is for postgresql9.6.3.

I think we should use postgreql version instead of photon version.

as we may have different PostgreSQL version with same photon os version in the future.

I can move the magic number 9.6.3 to a central place like the Makefile.

and change it on the run , just like __version__ in the docker-compose.tpl,  
What do you think?